Runtime/Plataforms/AndroidBuildPlatform.cs

```diff
@@ -0,0 +1,121 @@
+                PlayerSettings.Android.keystorePass = androidBuildPlatform.KeyStoreSettings.KeystorePassword;
+                PlayerSettings.Android.keyaliasName = androidBuildPlatform.KeyStoreSettings.AliasName;
+                PlayerSettings.Android.keyaliasPass = androidBuildPlatform.KeyStoreSettings.AliasPassword;
+            }
+            
+        }
+
+        public static void MakeDirectory(string relativePath)
+        {
+            var path = GetProjectPath() + relativePath;
+            if (Directory.Exists(path)) return;
+            Directory.CreateDirectory(path);
+        }
+
+        public override void OnBuild()
+        {
+            //PackageName
+            if (UsePackageName && !string.IsNullOrEmpty(PackageName))
+            {
+                PlayerSettings.SetApplicationIdentifier(BuildTargetGroup.Android, PackageName);
+            }
+
+            //Version
+            if (!UnityEditorInternal.InternalEditorUtility.isHumanControllingUs)
+            {
+                Version(out var gameVersion, out var bundleVersion);
+                PlayerSettings.bundleVersion = gameVersion;
+                PlayerSettings.Android.bundleVersionCode = bundleVersion;
+            }
+
+            //SetScriptDefinitions
+            if (UseCustomScriptDefinitions)
+            {
+                PlayerSettings.SetScriptingDefineSymbols(
+                    UnityEditor.Build.NamedBuildTarget.Android,
+                    ScriptDefinitions.Definitions
+                );
+            }
+
+            //Backend
+            PlayerSettings.SetScriptingBackend(BuildTargetGroup.Android, ScriptingBackend);
+
+            //Keystore Config
+            SetAndroidSignCredentials(buildScript);
+
+            if (BuildAAB)
+            {
+                //Make
+                const string subPathAab = "Build/Android/AAB/";
+                MakeDirectory(subPathAab);
+
+                var buildOptions = new BuildPlayerOptions
+                {
+                    scenes = GetScenes(),
+                    locationPathName = subPathAab + buildScript.GameName.ToLower().Replace(" ","_") + "_" + PlayerSettings.bundleVersion.Replace(".","_") + ".aab",
+                    target = BuildTarget.Android,
+                    options = GetOptions()
+                };
+
+                EditorUserBuildSettings.buildAppBundle = true;
+                ProcessReport(BuildPipeline.BuildPlayer(buildOptions));
+            }
+
+            if (BuildAPK)
+            {
+                //Make
+                const string subPathAPK = "Build/Android/APK/";
+                MakeDirectory(subPathAPK);
+
+                var buildOptions = new BuildPlayerOptions
+                {
+                    scenes = GetScenes(),
+                    locationPathName = subPathAPK + buildScript.GameName.ToLower().Replace(" ","_") + "_" + PlayerSettings.bundleVersion.Replace(".","_") + ".apk",
```

Comment:
breaking changes: new name will cause issues to people that are current using this package. Make this change a boolean option on this platform instead.